_sass/button.scss

```diff
@@ -7,7 +7,8 @@
     display: inline-block;
     font-weight: 400;
     font-family: inherit;
-    border-radius: $BORDER_RADIUS;
+    border-radius: 2px;
```

Comment:
It's better to define & change the value of border-radius in https://github.com/ChampionFX/binary-style/blob/gh-pages/_sass/constants.scss
So it will change all over the places